scss/_variables.scss

```diff
@@ -38,36 +38,151 @@ $roboto-light: Roboto-Light, $emojifonts, 'Helvetica Neue', Arial, Helvetica, No
+  --clr-global-a: #2090ea;
+  --clr-global-bg: #fff;
+  --clr-global-text: #070c14;
+  --clr-map-overlay-bg: #fff;
+  --clr-message-status-icon: #a4a6a9;
+  --clr-message-status-icon-sending: #62656a;
+  --clr-message-buttons: #8b8e91;
+  --clr-message-buttons-hover: #070c14;
+  --clr-message-padlock-outgoing: #4caf50;
+  --clr-message-padlock-incomming: #a4a6a9;
+  --clr-message-metadata-image-no-caption: #fff;
+  --clr-message-text: #070c14;
+  --clr-message-text-underline: #070c14;
+  --clr-message-generic-attachment-icon-extension: #070c14;
+  --clr-message-generic-attachment-filename: #070c14;
+  --clr-message-generic-attachment-filesize: #070c14;
+  --clr-message-attachment-container-bg: #ffffff;
+  --clr-message-attachment-filename-incomming: #ffffff;
+  --clr-message-attachment-filesize-incomming: #ffffff;
+  --clr-message-metadata-date: #62656a;
+  --clr-message-metadata-date-incomming: rgba(#ffffff, 0.7);
+  --clr-message-video-play: #2090ea;
+  --clr-message-video-overlay-circle: #ffffff;
+  --clr-message-author: #ffffff;
+  --clr-scrollbar-thumb: #666666;
+  --clr-scrollbar-thumb-hover: #606060;
 }
 
-$color-light-10: #eeefef; // only used by attachment downloadbutton :hover
+// Theming Colors
+/* Global */
+$color-global-a: var(--clr-global-a);
+$color-global-bg: var(--clr-global-bg);
+$color-global-text: var(--clr-global-text);
 
-// A few layout variables used cross-file
+$color-scrollbar-thumb: var(--clr-scrollbar-thumb);
+$color-scrollbar-thumb-hover: var(--clr-scrollbar-thumb-hover);
+
+/* Misc */
+$color-contact-default-label: var(--clr-avatar-label);
+$color-contact-list-item: var(--clr-contact-list-item);
+
+$color-bp3-dialog-bg: var(--clr-bp3-dialog-bg);
+$color-bp3-dialog-card-bg:var(--clr-bp3-dialog-card-bg);
+$color-bp3-dialog-header:var(--clr-bp3-dialog-header-bg);
+$color-bp3-heading:var(--clr-bp3-heading);
+$color-bp3-button-text:var(--clr-bp3-button-text);
+$color-bp3-button-bg:var(--clr-bp3-button-bg);
+
+$color-map-overlay-bg: var(--clr-map-overlay-bg);
+
+/* Message Composer */
+$color-composer-emoji-btn: var(--clr-composer-btn);
+$color-composer-file-btn: var(--clr-composer-btn);
+
+/* Chat List (conversation list) */
+$color-conversation-list-has-unread: var(--clr-conversation-list-unread-count-bg);
+$color-conversation-list-unread-count-bg: var(--clr-conversation-list-unread-count-bg);
+$color-conversation-list-unread-count-label: var(--clr-conversation-list-unread-count-label);
+$color-conversation-list-message-error: var(--clr-error);
+$color-conversation-list-avatar-text: var(--clr-avatar-label);
+$chatlist-isgroup-icon: var(--clr-global-text);
+
+/* Message Bubble */
+$color-message-author: var(--clr-message-author);
+$color-message-attachment-container-bg: var(clr-message-attachment-container-bg);
+$color-message-attachment-filename-incomming: var(--clr-message-attachment-filename-incomming);
+$color-message-attachment-filesize-incomming: var(--clr-message-attachment-filesize-incomming);
```

Comment:
incoming only needs one `m`

Reply:
Maybe you want to have different color shades for filename and filesize... but for this one we might get away with setting opacity for this effect, then we only need one var and if message-text-collor == filename-color == filesize-color we don't need any extra var for it.